crypto/fipsmodule/cipher/internal.h

```diff
@@ -73,12 +74,44 @@ extern "C" {
 // EVP_CIPH_MODE_MASK contains the bits of |flags| that represent the mode.
 #define EVP_CIPH_MODE_MASK 0x3f
 
+// Set of EVP_AEAD->aead_id identifiers, zero is reserved as the "unknown"
+// value since it is the default for a structure.
```

Comment:
Should we add a note about what we talked about offline? That is, something along the lines of: "Implementations of the same algorithms should likely have the same identifier. For example, machine-optimised versions should have the same identifier".